src/scenes/Activities/scenes/EventHistory/index.js

```diff
@@ -218,6 +232,10 @@ class EventHistory extends React.Component {
   }
 
   getEventURL(topicAddress, oracleAddress) {
+    if (!oracleAddress) {
```

Comment:
prefer to check if `!_.isEmpty(topicAddress)` instead of just assuming topicAddress is there.